pkg/trace/agent/agent.go

```diff
@@ -32,6 +32,9 @@ const (
 	// tagHostname specifies the hostname of the tracer.
 	// DEPRECATED: Tracer hostname is now specified as a TracerPayload field.
 	tagHostname = "_dd.hostname"
+
+	// tagErrorTrackingMode is set when the agent is configured to send only ET-eligible spans
+	tagErrorTrackingMode = "_dd.error_tracking.only"
```

Comment:
there are already some tags under `_dd.error_tracking`, I thought tagging along was a good idea.
This is invisible to the users anyway.